src/components/backup/BackupCloudStep.js

```diff
@@ -199,8 +199,8 @@ export default function BackupCloudStep() {
       password.length >= cloudBackupPasswordMinLength &&
       passwordFocused
     ) {
-      const passInfo = zxcvbn(password);
-      switch (passInfo.score) {
+      const passInfo = passwordStrength(password);
```

Comment:
did u check that the default strength validation matches (more o less) the previous behavior?  Remember cloudBackupPasswordMinLength is currently set to 8




<img width="487" alt="Screen Shot 2023-02-27 at 1 12 55 PM" src="https://user-images.githubusercontent.com/1247834/221648292-5cfbd4d7-e4db-4edb-9544-a050c386698d.png">

Reply:
yeah we are good with just doing it this way, length is only 1 factor. this will only impact new users and changing cloudBackupPasswordMinLength would fuck with our validation for existing users who try to restore